src/zimCheck.cpp

```diff
@@ -409,6 +413,9 @@ int main (int argc, char **argv)
                 getLinks( it -> getData() , &links);
                 for(unsigned int i = 0; i < links.size(); i++)
                 {
+                    // TODO: Code in this loop is really buggy, see #32
+                    continue;
```

Comment:
@mgautierfr I'm not going to fix all bugs in the SW with this PR. This PR exists to fix the wrong external dependences detection. Here the pb is in the internal links check which was - and still - is wrong. This is different. The `continue` is there only to avoid the app to `segfault` (which is new after the fix I have made).